flowcore/src/main/java/com/raizlabs/android/dbflow/sql/QueryBuilder.java

```diff
@@ -164,6 +164,29 @@ public QueryClass appendNotEmpty(String string) {
         return castThis();
     }
 
+    /**
+     * Returns quoted string.
+     *
+     * @param string The string to quote
+     * @return Quoted string
+     */
+    public static String quote(String string) {
+        if (string.compareTo("*") == 0)
```

Comment:
so it can become:

```
public static String quote(String string) {
  return "*".equals(string) ? string : "`" + string.replace(".", "`.`") + "`";
}

```